src/main/features/eligibility/views/not-eligible.njk

```diff
@@ -52,6 +57,12 @@
         <p>{{ t('You need to use a paper form if you want ' + externalLink('help paying your court fees', feesHelpUri ) + '.') | safe }}</p>
         <p>{{ t(externalLink('Download a paper form', n1FormUrl) + ', complete it and return it to make your claim.') | safe }}</p>
         {{ paperFormAddress() }}
+      {% elseif reason === NotEligibleReason.HELP_WITH_FEES_REFERENCE %}
+        <p>You can either:</p>
+      <ul class="list list-bullet">
+        <li>{{ t(externalLink('apply for Help With Fees ', feesHelpUri, '', false ) + ' and make a claim using a Help With Fees number') | safe }}</li>
+        <li>{{ internalLink( t('continue making this claim '), EligibilityPaths.startPage.uri) | safe }} {{ t(' and pay a fee based on the amount you claim' ) }}</li>
```

Comment:
might get an extra space between link and text, please double check once

Reply:
Good catch. Fixed.